ampligraph/evaluation/protocol.py

```diff
@@ -482,11 +483,11 @@ def evaluate_performance(X, model, filter_triples=None, verbose=False, strict=1,
 
     verbose : bool
         Verbose mode
-    strict : int
-        Strict mode. If 1 then any unseen entity will cause a RuntimeError.
-        If 0 then triples containing unseen entities will be filtered out.
-        If -1 then no filtering is done. One can use this mode, to skip filtering of unseen entities, if
-        train_test_split_unseen was used to create the splits.
+    filter_unseen : bool
+        If True then any unseen entity will be filtered out, if False it will cause a RuntimeError.
+        One can set this to False, to skip filtering of unseen entities, for eg, if train_test_split_unseen 
```

Comment:
Can you change this to:

"This can be set to False to skip filtering of unseen entities if ```train_test_split_unseen()``` was used to split the original dataset. "